ui/apps/platform/src/constants/entityTypes.ts

```diff
@@ -97,6 +97,23 @@ export const searchCategories: Record<string, SearchCategory> = {
     SUBJECT: 'SUBJECTS',
 };
 
+// Replace searchCategories value (not key) for flat data model.
+export function flattenImageSearchCategory(
```

Comment:
Nit: the naming here makes sense, but when reading this my mind immediately goes to "flatten a multi-dimensional array". What about `convertToFlatImageSearchCategory` or similar?

Reply:
Same with the functions in searchFilterConfig.ts

Reply:
Indeed, good eyes!

Explanation, not excuse, I bent over (too far) backward to be consistent with verb in feature flag.

For our future selves to consider, adjective or noun allows more direct relationship with derived names.